Gemfile

```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
```

Comment:
- [ ] Kindly fix this error in the console
![image](https://github.com/M-Anwar-Hussaini/Greeting/assets/24703171/ec417d7d-617e-4ea5-af96-4e48741bbda6)
I got this error and I didn't get any greeting message on the page.

Reply:
Hi @gemmen29.
Thanks for reviewing my project. in the ULR, instead of typing`http://127.0.0.1:3000/` kindly change it to: `http://localhost:3000/`. It will work.

Reply:
## the `http://localhost:3000/` URL works.

![image](https://github.com/M-Anwar-Hussaini/Greeting/assets/131291114/e6097c1f-6b4a-4677-a8df-09615136ca17)

## the `http://localhost:3000/` URL doesn't work work.

![image](https://github.com/M-Anwar-Hussaini/Greeting/assets/131291114/85c379f3-cd58-47ec-b63a-2e46f88876cd)

## The reason is because fetching the data from the `fetch` function in the following part of the code in line #9: 

![image](https://github.com/M-Anwar-Hussaini/Greeting/assets/131291114/37529df8-2426-41ef-86b0-06f0c400e496)
@gemmen29

Reply:
Hello @M-Anwar-Hussaini Thanks for clarification.